src/sources/audio.test.ts

```diff
@@ -21,5 +21,25 @@ describe('Sources', () => {
         expect(newFingerprint).toBe(newFingerprint)
       }
     })
+
+    it('returns a stable value', async () => {
+      const result = getAudioFingerprint()
+
+      if (isUnsupportedBrowser()) {
+        return
+      } else {
+        if (typeof result !== 'function') {
+          throw new Error('Expected to be a function')
+        }
+
+        const first = await result()
+        const second = await result()
+        expect(second).toBe(first)
```

Comment:
All calls of the same `result` function return literally the same promise, so this assertion can't fail. Therefore `getAudioFingerprint` should be called twice instead.

Reply:
Good catch, updated. Thanks!